eng/templates/Azure.ResourceManager.Template/.template.config/template.json

```diff
@@ -54,19 +54,43 @@
       "defaultValue": "",
       "replaces": "SwaggerVersionTag"
     },
+    "TypeSpecDirectory": {
+      "type": "parameter",
+      "datatype":"text",
+      "isRequired": false,
+      "description": "TypeSpecDirectory under sdk/providername.",
+      "defaultValue": "TypeSpecDirectory",
+      "replaces": "TypeSpecDirectory"
+    },
+    "SpecOrg": {
+      "type": "parameter",
+      "datatype":"text",
+      "isRequired": false,
+      "description": "the organization name of Azure spec repo.",
+      "defaultValue": "Azure",
+      "replaces": "SpecOrg"
+    },
+    "CommitId": {
+      "type": "parameter",
+      "datatype":"text",
+      "isRequired": false,
+      "description": "the commit id of Azure spec repo.",
+      "defaultValue": "CommitId",
+      "replaces": "CommitId"
+    },
     "includeCI": {
       "type": "parameter",
       "datatype": "bool",
       "isRequired": false,
       "description": "The optional parameter specifies whether generate related pipline ci.yml and test-resources.json in parent folder.",
       "defaultValue": "false"
     },
-    "ProviderShortNameLowercase": {
+    "LowercaseProviderShortName": {
```

Comment:
what is the rename for?

Reply:
ProviderShortNameLowercase  and  ProviderShortName  have the same beginning which can cause the ProviderShortNameLowercase  substitude with ProviderShortName + Lowercase